lib/dynamodb.js

```diff
@@ -3,9 +3,11 @@
 var Batch = require("./batch");
 var Transact = require("./transact");
 
-var DynamodbFactory = function ($config) {
-  return new DynamoDB($config);
+function createDynamoOopClient(dynamoDbClient) {
+  return new DynamoDB(dynamoDbClient);
 };
+
+var DynamodbFactory = createDynamoOopClient;
```

Comment:
```suggestion
/**
 * Minh recommand you to use createDynamoOopClient since 2.0.10
 */
var DynamodbFactory = createDynamoOopClient;
```

Just joking 😆 , but I feel like this will help ppl to migrate.

Reply:
Ah haha good idea

I've prepared PR to migrate it already, e.g. this 😛 https://github.com/TheAthletic/theathletic-apollo-express/pull/6012